ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/Exif.java

```diff
@@ -237,11 +237,15 @@ public Builder setUserComment(final String userComment) {
 
         @NonNull
         public Builder setOrientationFromDegrees(final int degrees) {
-            final byte[] bytes = new byte[1];
-            bytes[0] = (byte) rotationToExifOrientation(degrees);
-            final TiffOutputField orientationOutputField = new TiffOutputField(
-                    TiffTagConstants.TIFF_TAG_ORIENTATION, FieldType.SHORT, 1, bytes);
-            mIfd0Directory.add(orientationOutputField);
+            try {
+                final short orientation = rotationToExifOrientation(degrees);
+                final byte[] bytes = FieldType.SHORT.writeData(orientation,
+                        mTiffOutputSet.byteOrder);
+                final TiffOutputField orientationOutputField = new TiffOutputField(
```

Comment:
Weren't TIFF's unavailable in Android? Also, are you about leaving the catch closure empty?

Reply:
Exif uses Tiff tag information format for compatibility `Related  attribute  information  for  both  compressed  and  uncompressed  files  is  stored  in  the  tag  information  format  defined in TIFF Rev. 6.0` (Page 2 in http://www.exif.org/Exif2-2.PDF)

That exception is thrown by the `FieldType.SHORT.writeData()` method in case the type is not a short, which in this case cannot happen. I decided to ignore it.